dojo/settings/settings.dist.py

```diff
@@ -1091,6 +1093,8 @@ def saml2_attrib_map_format(dict):
     'Hadolint Dockerfile check': True,
     'Semgrep JSON Report': True,
     'Generic Findings Import': True,
+    'Edgescan Scan': True,
+    'Edgescan API Scan': True,
```

Comment:
It seems that loading from the API and getting data from a file load the same format/data.
Should we consider having only one scan_type?

Reply:
Fixed in 1bbd69a.